src/lib/schematics/utils/devkit-utils/ast-utils.ts

```diff
@@ -1,10 +1,5 @@
-/**
- * @license
- * Copyright Google Inc. All Rights Reserved.
- *
- * Use of this source code is governed by an MIT-style license that can be
- * found in the LICENSE file at https://angular.io/license
- */
+/* tslint:disable */
```

Comment:
why are we removing the license on these files if we added it everywhere else?

Reply:
Good catch. I accidentally removed them. Re-added and also uses `Google LLC` instead of `Google Inc.`